### ISSUE TITLE
chai-vague-errors should check 2nd expect arg for an error description

### ISSUE THREAD

**Issue by philipbulley** [Microsoft/tslint-microsoft-contrib#207]
It's possible to [specify a custom error message](http://chaijs.com/guide/styles/#expect) via the second arg to `expect()`

``` js
// AssertionError: topic [answer]: expected 43 to equal 42.
expect(43, 'topic [answer]').to.equal(42);
```

But `chai-vague-errors` produces `Found chai call with vague failure message. Please add an explicit failure message`

With the 2nd arg messages, we can continue to use the fluent chai syntax:

``` js
expect(something, 'expected something to have occurred').to.be.true;
```

without having to resort to:

``` js
expect(something).to.equal(true, 'expected something to have occurred');
```

So, should `chai-vague-errors` accommodate 2nd arg messages?

**Comment by HamletDRC** [comment]
Correct. We should support this but do not yet. 

I think the rule should be coded to force one style or the other, but not have mixing both in the same project.
